modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/commands/open_confirm_modal.js

```diff
@@ -27,7 +27,6 @@ const openConfirmModal = ({message, onConfirm, status, title}) => {
 					},
 				},
 			],
-			center: true,
```

Comment:
[opinionated] I am not sure why we chose center alignment. Confirm modal looks wrong to me when it is centered, inconsistent with the rest of DXP.

Before PR:
![image](https://github.com/liferay-frontend/liferay-portal/assets/5435511/bc706237-f105-4538-a5ca-be1ac67c5d63)

After PR:
![image](https://github.com/liferay-frontend/liferay-portal/assets/5435511/af4481a0-9d31-482d-aef3-cac5678bd028)